libexec/scanner/scanner.c

```diff
@@ -51,6 +51,8 @@
 #include <tsd/strutil.h>
 #include <tsd/percent.h>
 
+#define SCANLIMIT 10000
```

Comment:
Please make this a variable rather than a macro so we can more easily make it configurable later. We might even want to move it into `struct scanpath`.